src/ui/atoms/textField/TextField.tsx

```diff
@@ -20,44 +20,24 @@ export type TextFieldProps = InputBaseProps & {
    * If true, the TextField will take 100% of its parent's size.
```

Comment:
Perhaps it would be necessary to standardize the comments which explain the properties by calling either TextField or input, because both are present: "The size of the input field." / "If true, the TextField will take ..."